src/applications/static-pages/static-pages-entry.js

```diff
@@ -117,13 +104,7 @@ createHigherLevelReviewApplicationStatus(
   widgetTypes.HIGHER_LEVEL_REVIEW_APP_STATUS,
 );
 
-createApplicationStatus(store, {
-  formId: VA_FORM_IDS.FORM_21P_530,
-  applyHeading: 'How do I apply?',
-  additionalText: 'You can apply online right now.',
-  applyText: 'Apply for burial benefits',
-  widgetType: widgetTypes.BURIALS_APP_STATUS,
-});
```

Comment:
This doesn't have an `applyLink` like the others do, so I've removed that from the `burials` status widget as well.